chesster.js

```diff
@@ -1169,3 +1177,71 @@ controller.hears([
         }
     });
 });
+
+
+/* Scheduling */
+
+// reply to any incoming message
+controller.on('ambient', function(bot, message) {
+    bot_exception_handler(bot, message, function(){
+        var channel = channels.byId[message.channel];
+        var scheduling_options = config.scheduling[channel.name];
+        if (!scheduling_options) {
+            return;
+        } 
+        try {
+            var results = spreadsheets.parse_scheduling(message.text, {
+                offset_hours: scheduling_options.offset_hours
+            });
+            var white = users.getByNameOrID(results.white);
```

Comment:
This operation seems unnecessary. if we only want the name, then can we just return the name, or rather, can we just access the nae in update_schedule?

Not that important, but simplifies this slightly.